src/json_rpc_client.rs

```diff
@@ -51,24 +49,6 @@ pub(crate) fn run(last_config: Option<PathBuf>, cmd: CliCommand) -> Result<(), f
                 println!("Block for epoch #{} had digest {}", epoch, hash);
             }
 
-            Ok(())
-        }
-        CliCommand::GetOutput {
-            config,
-            output_index,
-        } => {
-            let config = config.or(last_config);
-            let mut stream = start_client(config)?;
-            let output_pointer = OutputPointer::from_str(&output_index)?;
-            let request_payload = serde_json::to_string(&output_pointer)?;
-            let request = format!(
-                r#"{{"jsonrpc": "2.0","method": "getOutput", "params": [{}], "id": "1"}}"#,
-                request_payload,
-            );
-            let response = send_request(&mut stream, &request)?;
```

Comment:
Instead of removing the getOutput command, we can just leave an "unimplemented" warning

             //let response = send_request(&mut stream, &request)?;
             let response = "Unimplemented";